pages/common/docker-build.md

```diff
@@ -15,6 +15,10 @@
 
 `docker build --tag {{name:tag}} .`
 
+- Build a docker image with no build context:
+
+`docker build --tag {{name:tag}} - < {{Dockerfile}}`
```

Comment:
Seems specific to *nix shells (it's not a valid on Windows): https://docs.docker.com/engine/reference/commandline/build/#text-files

Reply:
Should this example be after `docker build {{github.com/creack/docker-firefox}}`? In fact, the 2 first examples are related to context: current dir + URL.

Reply:
> Seems specific to *nix shells (not valid on Windows):

Ahh shoot, yeah that slipped my mind.
I'll update this later to accommodate that.

> Should this example be after docker build {{github.com/creack/docker-firefox}}?

Sorry, I don't understand what you mean? I put the example 4th just because the 3rd is the standard way to use a Dockerfile with the current directly as the build context, and this command is the standard way to use a Dockerfile to have n build context at all.

Reply:
Nope @nicokosi, that syntax is not specific to *nix shells - it works on Windows too. Not sure on Powershell, but powershell is not the default command line on Windows anyway.

Reply:
Windows PowerShell doesn't like it 😞

>The '<' operator is reserved for future use.

But that doesn't matter.